awx/main/models/jobs.py

```diff
@@ -680,6 +680,14 @@ def display_artifacts(self):
             return "$hidden due to Ansible no_log flag$"
         return artifacts
 
+    @property
+    def can_run_containerized(self):
+        return any([ig for ig in self.preferred_instance_groups if ig.is_containerized])
+
+    @property
+    def is_containerized(self):
+        return self.instance_group and self.instance_group.is_containerized
```

Comment:
Would wrap in a bool() cast. If self.instance_group is None and self.instance_group.is_containerized is None then the result return will be None vs. False.